Cargo.toml

```diff
@@ -124,18 +124,16 @@ http-body-util = "0.1"
 hyper = { version = "1.1", features = ["http1", "client"] }
 hyper-util = { version = "0.1.12", features = ["http1", "client", "client-legacy", "client-proxy", "tokio"] }
 h2 = { version = "0.4", optional = true }
-once_cell = "1.18"
 log = "0.4.17"
-mime = "0.3.16"
 percent-encoding = "2.3"
 tokio = { version = "1.0", default-features = false, features = ["net", "time"] }
 tower = { version = "0.5.2", default-features = false, features = ["timeout", "util"] }
 tower-http = { version = "0.6.5", default-features = false, features = ["follow-redirect"] }
 pin-project-lite = "0.2.11"
-ipnet = "2.3"
```

Comment:
Code search shows that we don't use any API from this anymore: https://github.com/search?q=repo%3Aseanmonstar%2Freqwest+ipnet&type=code